app/src/androidTest/java/app/cybrid/demoapp/ui/tradeFlow/IdentityVerificationActivityTest.kt

```diff
@@ -60,7 +63,7 @@ class IdentityVerificationActivityTest {
         val loadingText = "Checking Identity..."
         compose.onNodeWithText(loadingText).assertIsDisplayed()
 
-        compose.waitUntilDoesNotExist(hasText(loadingText), 10_000L)
+        compose.waitUntilDoesNotExist(hasText(loadingText), 15_000L)
```

Comment:
Did we hit the timeout? Is that why this got changed?

Reply:
Yeas the problem is the emulator in CI sometimes is so slow, for that reason I added other 5 seconds